cmd/up.go

```diff
@@ -54,6 +56,7 @@ var upCmd = &cobra.Command{
 }
 
 func init() {
+	upCmd.Flags().BoolVar(&UpInsecureRepo, "insecure-repository", false, "Use Insecure Repository")
```

Comment:
we will have to clean up flags a bit, so they are defined in one place, but that should be solved in different PR